src/components/Login/components/authError/AuthError.js

```diff
@@ -0,0 +1,50 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { FormattedMessage } from 'react-intl';
+import classNames from 'classnames';
+
+import styles from './AuthError.css';
+
+export default class AuthError extends Component {
```

Comment:
As far as I understand this component is an error container. Maybe we should rename it? AuthErrorsContainer or ErrorsPlaceholder?